temba/settings.py.prod

```diff
@@ -408,3 +407,27 @@ USER_RECOVER_MAX_ATTEMPTS = env.int("USER_RECOVER_MAX_ATTEMPTS", default=5)
 
 # Channel that should not be displayed in the old design
 OLD_DESIGN_EXCLUDED_CHANNELS_CODES = env.list("OLD_DESIGN_EXCLUDED_CHANNELS_CODES", default=["TG", "WWC", "IG"])
+
+DISABLE_OIDC = env.bool("DISABLE_OIDC", default=False)
+if DISABLE_OIDC:
+    REST_FRAMEWORK["DEFAULT_AUTHENTICATION_CLASSES"].remove(
+        "mozilla_django_oidc.contrib.drf.OIDCAuthentication"
+    )
+
+    middleware_list = list(MIDDLEWARE)
+    middleware_list.remove("mozilla_django_oidc.middleware.SessionRefresh")
+    MIDDLEWARE = tuple(middleware_list)
+
+    installed_apps_list = list(INSTALLED_APPS)
+    installed_apps_list.remove("mozilla_django_oidc")
+    installed_apps_list.remove("weni.auth")
+    INSTALLED_APPS = tuple(installed_apps_list)
+
+    LOGGING["loggers"].pop("mozilla_django_oidc")
+    LOGGING["loggers"].pop("weni_django_oidc")
+
+    authentication_backend_list = list(AUTHENTICATION_BACKENDS)
+    authentication_backend_list.remove("weni.auth.backends.WeniOIDCAuthenticationBackend")
```

Comment:
whats the diffence between the `remove` and `pop`, I mean, why in some cases are you using `pop` and others using `remove`?

Reply:
`pop` is being used in dictionaries, and `remove` in lists